builder/azure/arm/builder.go

```diff
@@ -318,6 +320,11 @@ func (b *Builder) Run(ctx context.Context, ui packersdk.Ui, hook packersdk.Hook)
 		return nil, errors.New("Build was halted.")
 	}
 
+	if _, ok := b.stateBag.GetOk(constants.ArmCaptureTemplate); !ok {
```

Comment:
I know checking for images is the way the Amazon and GCP plugins handle this but what if we just checked if we're supposed to be skipping create image, and otherwise return, that way if in the future there's some sort of bug where we got to here in the code without images (when not skipping), we wouldn't just return nil.
```suggestion
	if b.config.SkipCreateImage {
```
Does that make sense? What do you think?

Reply:
Yep, that sounds good, I wavered on using our flag vs matching others, good to have an opinion on it :). I'll commit and then rebase it quick. Thanks!